Sources/SwiftDocC/SwiftDocC.docc/Resources/LinkableEntities.json

```diff
@@ -16,6 +16,7 @@
             },
             "LinkDestinationSummary": {
                 "type": "object",
+                "additionalProperties": false,
```

Comment:
Setting `additionalProperties` to false allows for better/stricter validation of the spec. Otherwise a spec validator will allow arbitrary additional properties without throwing an error. Without this, the `declarationFragments` vs `fragments` misspelling wouldn't be caught.